### DISCUSSION TITLE
Connect figma tokens / variables to primeReact library

### DISCUSSION THREAD

**Discussion by hahajska** [primefaces/community#1386] [Figma UI Kit]
Hey,
could you please confirm that I understand everything around tokens, its syncing with primerReact library correctly?

Right now,:

it is not possible to connect figma tokens to primeReact library
it is not possible to use json with tokens / varibles in primeReact library

Thank you

**Comment by onursenture** [comment]
We currently do not have official support for this feature. However, we are actively working on it.
